components/src/md3/MobileStepper/MobileStepper.tsx

```diff
@@ -0,0 +1,193 @@
+import React from 'react';
+import { View, StyleSheet, StyleProp, ViewStyle, ViewProps } from 'react-native';
+import { MD3Theme, ProgressBar, useTheme } from 'react-native-paper';
+import * as Colors from '@brightlayer-ui/colors';
+import { $DeepPartial } from '@callstack/react-theme-provider';
+import { Body1 } from '../../core/typography';
+import { getPrimary500 } from '../../core/utility/shared';
+
+const makeStyles = (
+    props: MobileStepperProps,
+    theme: MD3Theme
+): StyleSheet.NamedStyles<{
+    root: ViewStyle;
+    stepperContainer: ViewStyle;
+    circle: ViewStyle;
+    filled: ViewStyle;
+    progressBar: ViewStyle;
+    text: ViewStyle;
+}> =>
+    StyleSheet.create({
+        root: {
+            alignItems: 'center',
+            justifyContent: 'center',
+            flexDirection: 'row',
+            margin: 16,
+        },
+        stepperContainer: {
+            flexDirection: 'row',
+            flex: 1,
+            alignItems: 'center',
+            justifyContent: 'center',
+        },
+        circle: {
+            height: 8,
+            width: 8,
+            borderRadius: 8,
+            marginHorizontal: 4,
+            overflow: 'hidden',
+            backgroundColor: props.inactiveColor || (theme.dark ? theme.colors.onBackground : Colors.gray[200]),
+        },
+        filled: {
+            backgroundColor: props.activeColor || (theme.dark ? theme.colors.primary : theme.colors.primary),
```

Comment:
This tertiary operator is not doing anything

Reply:
Removed tertiary operator from here, Thanks